Makefile

```diff
@@ -138,26 +138,27 @@ clean: assert-dom0 prep-dev ## Destroys all SD VMs
 .PHONY: test-prereqs
 test-prereqs: assert-dom0 ## Checks that test prerequisites are satisfied
 	@echo "Checking prerequisites before running test suite..."
-	test -e config.json || exit 1
-	test -e sd-journalist.sec || exit 1
+	test -e config.json || (echo "Ensure config.json is in this directory" && exit 1)
+	test -e sd-journalist.sec || (echo "Ensure sd-journalist.sec is in this directory" && exit 1)
+	which pytest coverage || (echo 'please install test dependencies with "sudo python3-pytest python3-pytest-cov"' && exit 1)
```

Comment:
There's a slight mistake in the text here: `sudo python3-pytest python3-pytest-cov` should be `sudo qubes-dom0-update python3-pytest python3-pytest-cov`